internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -189,6 +189,25 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster, awsCli
 			return errors.Wrapf(err, "failed to set %s", setValue)
 		}
 	}
+
+	if len(provisioner.params.EtcdManagerEnv) > 0 {
+		var override []string
+		var overrideIndex int
+		for key, val := range provisioner.params.EtcdManagerEnv {
+			override = append(override, "spec.etcdClusters[*].manager.env=")
```

Comment:
After thinking a bit about this, If this line is to create the `env` dictionary under `manager` it should be outside of the `for` loop. Maybe is working right now because declaring it with an empty value doesn't remove existing elements as you told me yesterday.

If this line is to create a new index under the `.manager.env`, you can disregard this comment.